package.json

```diff
@@ -13,12 +13,18 @@
   "engines": {
     "node": ">=4"
   },
+  "scripts": {
```

Comment:
You'll need husky as well to ensure this hook runs.

Reply:
@ianwremmel thank you, I have included husky in the lastest commit.